QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+```
+
+which should yield something like:
+
+```console
+namespace/kcp-edge-syncer-florin-1yi5q9c4 created
+serviceaccount/kcp-edge-syncer-florin-1yi5q9c4 created
+secret/kcp-edge-syncer-florin-1yi5q9c4-token created
+clusterrole.rbac.authorization.k8s.io/kcp-edge-syncer-florin-1yi5q9c4 created
+clusterrolebinding.rbac.authorization.k8s.io/kcp-edge-syncer-florin-1yi5q9c4 created
+secret/kcp-edge-syncer-florin-1yi5q9c4 created
+deployment.apps/kcp-edge-syncer-florin-1yi5q9c4 created
+```
+
+Optionally, check that the edge syncer pod is running:
+
+```console
+KUBECONFIG=$florin_kubeconfig kubectl get pods -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE                         NAME                                               READY   STATUS    RESTARTS   AGE
+kcp-edge-syncer-florin-1yi5q9c4   kcp-edge-syncer-florin-1yi5q9c4-77cb588c89-xc5qr   1/1     Running   0          12m
+kube-system                       coredns-565d847f94-92f4k                           1/1     Running   0          58m
+kube-system                       coredns-565d847f94-kgddm                           1/1     Running   0          58m
+kube-system                       etcd-florin-control-plane                          1/1     Running   0          58m
+kube-system                       kindnet-p8vkv                                      1/1     Running   0          58m
+kube-system                       kube-apiserver-florin-control-plane                1/1     Running   0          58m
+kube-system                       kube-controller-manager-florin-control-plane       1/1     Running   0          58m
+kube-system                       kube-proxy-jmxsg                                   1/1     Running   0          58m
+kube-system                       kube-scheduler-florin-control-plane                1/1     Running   0          58m
+local-path-storage                local-path-provisioner-684f458cdd-kw2xz            1/1     Running   0          58m
+
+``` 
+
+Now, let's onboard the `guilder` cluster:
+
+```shell
+kubectl kcp-edge prep-for-cluster --imw root:example-imw guilder env=prod
+```
+
+Apply the created edge syncer manifest:
+
+```shell
+KUBECONFIG=$guilder_kubeconfig kubectl apply -f guilder-syncer.yaml
+```
+
+
+### e. Create and deploy the Apache Server workload into florin and guilder clusters
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location objects (`florin` and `guilder`) created earlier, thus directing the workload to both edge clusters.
+
+In the `wmw-1` workspace create the following `EdgePlacement` object: 
+  
+```shell
+kubectl ws root:my-org:example-wmw
+kubectl apply -f examples/common-placement.yaml
+```
+
+Deploy the Apache HTTP Server workload. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
```

Comment:
(1) "Deploy" is a confusing word to use here, since the use here covers only getting the workload prescription into the WMW.  I would use wording more like "Put the common workload prescription into the WMW".

(2) The text here calls the reader's attention to something not shown ("the namespace label").  I think it would be good to include a display of the content of the file being used here (and in the other places where a file is used).

Reply:
Fixed